firefox-ios/Client/Frontend/Fakespot/FakespotViewController.swift

```diff
@@ -67,9 +65,7 @@ class FakespotViewController: UIViewController,
         label.text = .Shopping.SheetHeaderTitle
         label.numberOfLines = 0
         label.adjustsFontForContentSizeCategory = true
-        label.font = DefaultDynamicFontHelper.preferredFont(withTextStyle: .headline,
-                                                            size: UX.titleLabelFontSize,
-                                                            weight: .semibold)
+        label.font = FXFontStyles.Regular.headline.scaledFont()
```

Comment:
@cwzilla Regarding our conversation related to which headline type to use (either Bold or Regular styles). Which one would designers use for headline in general if they both are the same type?

Reply:
discussed that Regular is the way to go